### PR TITLE
Remove backports of py3 

### PR DESCRIPTION
**Pull request by emord**
@dimagi/py3 Haven't seen a PR for these (and thought they were probably a bit harder to find)

Was going to just remove the random.choices backport introduced in https://github.com/dimagi/commcare-hq/pull/23340

but when i searched I found the subprocess32 dependency I didn't know about

### CODE REVIEW

**Review by snopoke — APPROVED** [review]
Do they need to be added to `uninstall-requirements.txt`?

**Comment by emord** [comment]
Thanks. I forgot about that

File: corehq/sql_db/util.py [hunk 1]
```diff
@@ -1,28 +1,21 @@
...
+from random import choices
 
-from django.db import OperationalError
-
-from corehq.util.datadog.utils import load_counter_for_model
-
-try:
-    from random import choices
-except ImportError:
-    from corehq.sql_db.backports import choices
```
Comment (kaapstorm) [inline corehq/sql_db/util.py]: `corehq.sql_db.backports.choices()` can be killed too.

**Review by kaapstorm — APPROVED** [review]
Nice.

File: corehq/sql_db/util.py (hunk 1 above)
Comment (emord) [reply corehq/sql_db/util.py]: What do you mean? I removed the whole file in https://github.com/dimagi/commcare-hq/pull/25228/commits/2f47777a93937dd072e99a9e3f10372881ed250d

Comment (kaapstorm) [reply corehq/sql_db/util.py]: Sorry, I missed that.

Today I'm just going to blame GitHub:
image

**Pull request merged**